sdk/tests/tutorials/ibor/test_portfolio_group.py

```diff
@@ -0,0 +1,260 @@
+import unittest
+import uuid
+from datetime import datetime
+
+import pytz
+
+import lusid
+import lusid.models as models
+from lusidfeature import lusid_feature
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class GroupPortfolios(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+
+        cls.scopes_api = lusid.ScopesApi(api_client)
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        cls.portfolio_group_api = lusid.PortfolioGroupsApi(api_client)
+
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
```

Comment:
Why are we loading instruments? I don't think this tests needs instruments? Perhaps remove?